test.properties

```diff
@@ -2421,6 +2421,7 @@
         functional-smoke-tomcat90-mariadb106,\
         functional-smoke-tomcat90-mysql57-chrome100,\
         functional-smoke-tomcat90-mysql80,\
+        functional-smoke-tomcat90-mysql84,\
```

Comment:
To add the query for smoke mysql84 you can follow similar format here: https://github.com/joshitagaki/liferay-portal/blob/25e4b3b86fd3db8bcd9b25f187f19175968030b4/test.properties#L1760-L1769
this section of batch queries seem to be the default names